front/pages/api/w/[wId]/subscriptions/index.ts

```diff
@@ -43,29 +45,31 @@ async function handler(
   }
 
   switch (req.method) {
-    case "GET":
-      // Should return the list of featured plans
-      return apiError(req, res, {
-        status_code: 404,
-        api_error: {
-          type: "invalid_request_error",
-          message: "Not implemented yet.",
-        },
-      });
-
     case "POST":
-      const newPlan = await subscribeWorkspaceToPlan(auth, {
-        planCode: req.body.planCode,
-      });
-      res.status(200).json({ plan: newPlan });
-      return;
-
+      try {
+        const stripeCheckoutUrl = await subscribeWorkspaceToPlan(auth, {
+          planCode: req.body.planCode,
+        });
+        return res
+          .status(200)
+          .json({ success: true, checkoutUrl: stripeCheckoutUrl || undefined });
```

Comment:
I would return the plan here as well I think? it's clearer than `success:true` which serves no purpose. I would be in favour of removing `success:true` at least?